gnc/navigator_perception/exFAST_SparseStereo/GPL.txt

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
I recommend leaving the entire license with the code, we are technically legally obligated to.